assets/js/guides.js

```diff
@@ -1,4 +1,9 @@
 $(document).ready(function () {
+
+  if (window.location.href.indexOf('newsletter_success') !== -1) {
+    $("#newsletter-success").modal('show');
+  } 
```

Comment:
This check should be in `main.js` so it affects all pages, not just guides.

Reply:
Okay